hawkbit-runtime/hawkbit-update-server/src/main/resources/application-postgresql.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2015 Bosch Software Innovations GmbH and others.
```

Comment:
I think you're not a fellow of Bosch Software Innovations. So please change the title in order to give proper credit to either Copyright (c) 2020 Andrey Voronkov and others or Copyright (c) 2020 <YOUR COMPANY> and others. Depending on whether or not your employment contract stipulates that intellectual property developed by you is the property of your company